gui/default/syncthing/settings/advancedSettingsModalView.html

```diff
@@ -134,68 +138,73 @@ <h4 class="panel-title" tabindex="0" translate>Defaults</h4>
+              <div class="panel panel-default">
+                <div class="panel-heading" role="tab" id="advancedDefaultIgnoresHeading" data-toggle="collapse" data-parent="#advancedDefaults" href="#advancedDefaultIgnores" aria-expanded="false" aria-controls="advancedDefaultIgnores" style="cursor: pointer;">
+                  <h4 class="panel-title" tabindex="0" translate>Default Ignore Patterns</h4>
+                </div>
+                <div id="advancedDefaultIgnores" class="panel-collapse collapse" role="tabpanel" aria-labelledby="advancedDefaultIgnoresHeading">
+                  <div class="panel-body">
+                    <div class="form-horizontal">
+                      <div ng-repeat="(key, value) in advancedConfig.defaults.ignores" ng-if="inputTypeFor(key, value) != 'skip'" class="form-group">
+                        <label for="advancedDefaultIgnoresInput{{$index}}" class="col-sm-4 control-label">{{key | uncamel}}&nbsp;<a href="{{docsURL('users/config#config-option-defaults.ignores.')}}{{key | lowercase}}" target="_blank"><span class="fas fa-question-circle"></span></a></label>
+                        <div class="col-sm-8">
+                          <div ng-switch="key">
+                            <!-- Special case to preserve empty lines in multi-line input -->
+                            <textarea ng-switch-when="lines" id="advancedDefaultIgnoresInput{{$index}}" class="form-control" rows="5" ng-model="advancedConfig.defaults.ignores._lines" ng-model-options="{ getterSetter: true }"></textarea>
+                            <input ng-switch-default ng-if="inputTypeFor(key, value) == 'list'" id="advancedDefaultIgnoresInput{{$index}}" class="form-control" type="text" ng-model="advancedConfig.defaults.ignores[key]" ng-list />
+                            <input ng-switch-default ng-if="inputTypeFor(key, value) != 'list'" id="advancedDefaultIgnoresInput{{$index}}" class="form-control" type="{{inputTypeFor(key, value)}}" ng-model="advancedConfig.defaults.ignores[key]" />
+                          </div>
+                        </div>
                       </div>
                     </div>
                   </div>
-                </form>
+                </div>
               </div>
-            </div>
-
+            </form>
           </div>
         </div>
       </div>
-
     </div>
 
     <div class="modal-footer">
-      <button type="button" class="btn btn-primary btn-sm" ng-click="saveAdvanced()">
+      <button type="button" class="btn btn-primary btn-sm" ng-click="saveAdvanced()" ng-disabled="guiConfig.$invalid || optionsConfig.$invalid || ldapConfig.$invalid || advancedFolders.$invalid || advancedDevices.$invalid || advancedDefaults.$invalid || (guiConfig.$pristine && optionsConfig.$pristine && ldapConfig.$pristine && advancedFolders.$pristine && advancedDevices.$pristine && advancedDefaults.$pristine)">
```

Comment:
This is such a huge mouthful, might be worth moving into a utlity function? Though I don't remember the angularjs scopes, i.e. if we'd have access to `guiConfig` et. al. from such a utility function.

Reply:
Done. I've just removed the separate forms and wrapped everything in a one large form, so that the check is very short. There are no visual or functional changes as far as I've tested.